examples/test_cpp/test_cpp.vcxproj

```diff
@@ -88,7 +88,7 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <SDLCheck>true</SDLCheck>
+      <SDLCheck>false</SDLCheck>
```

Comment:
Is this still an issue? I remember this was from a broken VS version.

Reply:
I've seen it just few days ago - without this flag deprecation warnings (which we have for ccb methods) are treated as errors

Reply:
Ah okay fair enough